pkg/pyexp/python_adapter.go

```diff
@@ -55,7 +55,7 @@ func PyExecReq(jsonPayload string, p PyDepGetter) (ExecRequest, error) {
 	}
 
 	logger := funcr.New(func(prefix, args string) {
-		fmt.Println(prefix, args)
+		fmt.Println(prefix, args) //nolint:forbidigo // logger implementation
```

Comment:
This is actually okay to be printed since we use it from Python, so we don't have a logger. the `notlint` is a great addition!